.ci/update-json-specs.yml

```diff
@@ -0,0 +1,119 @@
+name: update-jsons-specs
+title: Update JSON specs
+
+scms:
+  default:
+    kind: github
+    spec:
+      user: '{{ requiredEnv "GIT_USER" }}'
+      email: '{{ requiredEnv "GIT_EMAIL" }}'
+      owner: elastic
+      repository: apm-agent-dotnet
+      token: '{{ requiredEnv "GITHUB_TOKEN" }}'
+      username: '{{ requiredEnv "GIT_USER" }}'
+      branch: main
+
+sources:
+  sha:
+    kind: file
+    spec:
+      file: 'https://github.com/elastic/apm/commit/main.patch'
+      matchpattern: "^From\\s([0-9a-f]{40})\\s"
+    transformers:
+      - findsubmatch:
+          pattern: "[0-9a-f]{40}"
+
+  cgroup_parsing.json:
+    kind: file
+    spec:
+      file: https://raw.githubusercontent.com/elastic/apm/main/tests/agents/json-specs/cgroup_parsing.json
+  service_resource_inference.json:
+    kind: file
+    spec:
+      file: https://raw.githubusercontent.com/elastic/apm/main/tests/agents/json-specs/service_resource_inference.json
+  span_types.json:
+    kind: file
+    spec:
+      file: https://raw.githubusercontent.com/elastic/apm/main/tests/agents/json-specs/span_types.json
+  sql_signature_examples.json:
+    kind: file
+    spec:
+      file: https://raw.githubusercontent.com/elastic/apm/main/tests/agents/json-specs/sql_signature_examples.json
+  sql_token_examples.json:
+    kind: file
+    spec:
+      file: https://raw.githubusercontent.com/elastic/apm/main/tests/agents/json-specs/sql_token_examples.json
+  w3c_distributed_tracing.json:
+    kind: file
+    spec:
+      file: https://raw.githubusercontent.com/elastic/apm/main/tests/agents/json-specs/w3c_distributed_tracing.json
+  wildcard_matcher_tests.json:
+    kind: file
+    spec:
+      file: https://raw.githubusercontent.com/elastic/apm/main/tests/agents/json-specs/wildcard_matcher_tests.json
+actions:
+  pr:
+    kind: "github/pullrequest"
+    scmid: default
+    spec:
+      automerge: false
+      draft: false
+      labels:
+        - "automation"
+      description: |-
```

Comment:
What do you think adding the `title` field so the [name of the PR](https://www.updatecli.io/docs/plugins/actions/github/#_parameters) is something handled also here? See